shell/client/shell.html

```diff
@@ -546,6 +546,11 @@ <h4 class="who-has-access">Who has access</h4>
     </div>
     {{#if completionState.clear}}
     <div class="button-container" role="presentation">
+      {{#with invitationExplanation}}
```

Comment:
Argh, Meteor moved the docs.  Now it's just http://docs.meteor.com/#/full/pkg_spacebars and I can't link to the particular fragment I wanted to.

The Meteor guide recommends using `{{#let}}` instead of `{{#with}}`, so let's go with that: http://guide.meteor.com/blaze.html#builtin-block-helpers